### ISSUE TITLE
pretrained_model

### ISSUE THREAD

**Issue by wangxinzhe123** [kenchan0226/keyphrase-generation-rl#22]
Hello, I have the following problems:No such file or directory: '[path_to_ml_pretrained_model]' when I run the code. I don't have pretrain model. Is this model trained by myself or downloaded from somewhere.Thank you very much！

**Comment by kenchan0226** [comment]
Hi, you need to use ml loss to train a model by yourself according to our instruction in "Train ML".

**Comment by wangxinzhe123** [comment]
I've already run the previous command. I will run it again to prevent problems in my running process. Thank you very much for your reply.

**Comment by kenchan0226** [comment]
If you have run the ML training, then you will have the ml trained models. You just need to set [path_to_ml_pretrained_model] to path of the ml trained model that has the lowest validation loss.
